amxmodx/newmenus.cpp

```diff
@@ -1076,6 +1076,112 @@ static cell AMX_NATIVE_CALL menu_setprop(AMX *amx, cell *params)
 	return 1;
 }
 
+// Gets a menu property
+// native menu_getprop(menu, prop, ...);
+static cell AMX_NATIVE_CALL menu_getprop(AMX *amx, cell *params)
+{
+	enum args { arg_numargs, arg_menu, arg_prop, arg_buffer, arg_bufferlen };
+
+	GETMENU(params[arg_menu]);
+
+	int paramsNum = params[arg_numargs] / sizeof(cell);
+
+	switch(params[arg_prop])
+	{
+		case MPROP_PAGE_CALLBACK: return pMenu->pageCallback;
+		case MPROP_SHOWPAGE: return pMenu->showPageNumber;
+		case MPROP_SET_NUMBER_COLOR:
+		{
+			if (paramsNum < arg_bufferlen)
+			{
+				LogError(amx, AMX_ERR_NATIVE, "Expected 4 parameters");
+				return 0;
+			}
+
+			set_amxstring(amx, params[arg_buffer], pMenu->m_ItemColor.chars(), params[arg_bufferlen]);
+			break;
+		}
+		case MPROP_PERPAGE:	return pMenu->items_per_page;
```

Comment:
I think here should a space instead of a tabulation.

Reply:
For some reason it displayed it as a space in my text editor. Fixed.